docs/ddl-of-carbondata.md

```diff
@@ -646,6 +647,68 @@ CarbonData DDL statements are documented here,which includes:
 
 ## TABLE MANAGEMENT  
 
+### DESCRIBE COMMAND
+
+Describe column of table and visualize its structure with child fields.
```

Comment:
Add sub-headings for Describe column and describe short. And divide syntax and example, similar to other commands

Reply:
Done